pkgs/tools/security/pgpdump/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "pgpdump-${version}";
+  version = "v0.31";
+
+  src = fetchurl {
```

Comment:
Thanks :+1: 

I fixed the version (I'm now prefixing the 'v' in `rev = "v${version}"` - hope thats ok) and I switched to `fetchFromGitHub`.